src/dde-file-manager-lib/views/computerviewitemdelegate.cpp

```diff
@@ -129,7 +129,7 @@ void ComputerViewItemDelegate::paint(QPainter *painter, const QStyleOptionViewIt
         painter->drawPixmap(option.rect.x() + leftmargin, option.rect.y() + topmargin, icon.pixmap(iconsize));
 
         painter->setFont(par->font());
-        painter->setPen(qApp->palette().color((option.state & QStyle::StateFlag::State_Selected) ? QPalette::ColorRole::HighlightedText : QPalette::ColorRole::Text));
```

Comment:
The following talk is from the designer:

````
The text color of selected state should be use QPalette::BrightText.
````

Please try it and upload the screenshot for both dark and light theme.

Reply:
![image](https://user-images.githubusercontent.com/10684972/178658528-ae195945-49c2-4fca-94e5-ea64987a40e0.png)

![image](https://user-images.githubusercontent.com/10684972/178658574-a5c166e5-8ae2-42fd-b85e-7fe0c9b0c4e3.png)

Reply:
Yes, it's right.